<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window2</class>
 <widget class="QDialog" name="Window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 178, 137);

background-color: rgb(255, 249, 166);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
background-color: rgb(239, 255, 213);
color: rgb(0, 0, 0);
font: 700 10pt &quot;Segoe UI&quot;;
font: 700 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Додати повідомлення&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Ім'я:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 152, 49);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Вік:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Consolas&quot;;
color: rgb(0, 22, 74);
background-color: rgb(170, 152, 49);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Стать:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Consolas&quot;;
color: rgb(0, 22, 74);
background-color: rgb(170, 152, 49);</string>
         </property>
         <item>
          <property name="text">
           <string>Хлопчик</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Дівчинка</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Кількість&lt;/p&gt;&lt;p align=&quot;center&quot;&gt; хороших вчинків:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 152, 49);
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Кількість &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;поганих вчинків&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 152, 49);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Consolas&quot;;
color: rgb(0, 22, 74);
background-color: rgb(170, 152, 49);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Тип подарунку:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Consolas&quot;;
color: rgb(0, 22, 74);
background-color: rgb(170, 152, 49);</string>
       </property>
       <item>
        <property name="text">
         <string>Їстівний</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Неїстівний</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <property name="styleSheet">
        <string notr="true">font: 700 9pt &quot;Consolas&quot;;
color: rgb(0, 22, 74);
background-color: rgb(170, 152, 49);</string>
       </property>
       <item>
        <property name="text">
         <string>Цукерки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Печиво</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Шоколад</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>Відправити</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
